tests/test_resolver.py

```diff
@@ -1,157 +1,281 @@
 # -*- coding: utf-8 -*-
 
-from os.path import join as pjoin
-from pathlib import Path
-from tempfile import TemporaryDirectory
+import os
+import shutil
 
-from tests.base import TestCase, assets, main, copy_of_directory
+from pathlib import (
+    Path
+)
+from unittest import (
+    mock
+)
+
+import pytest
+
+from tests.base import (
+    assets, 
+    main
+)
 
 from ocrd.resolver import Resolver
-from ocrd_utils import pushd_popd, initLogging
+from ocrd_utils import pushd_popd
+
+# set pylint once on module level
+# pylint: disable=protected-access
 
 METS_HEROLD = assets.url_of('SBB0000F29300010000/data/mets.xml')
 FOLDER_KANT = assets.path_to('kant_aufklaerung_1784')
+DATA_KANT = {'mets.xml': (os.path.join(FOLDER_KANT, 'data', 'mets.xml'), 'text/xml'),
+             'INPUT_0017.tif': (os.path.join(FOLDER_KANT, 'data', 'OCR-D-IMG', 'INPUT_0017.tif'), 'image/tiff'),
+             'INPUT_0020.tif': (os.path.join(FOLDER_KANT, 'data', 'OCR-D-IMG', 'INPUT_0020.tif'), 'image/tiff'),
+             'PAGE_0017_ALTO.xml': (os.path.join(FOLDER_KANT, 'data', 'OCR-D-GT-ALTO', 'PAGE_0017_ALTO.xml'), 'text/xml'),
+             'PAGE_0020_ALTO.xml': (os.path.join(FOLDER_KANT, 'data', 'OCR-D-GT-ALTO', 'PAGE_0020_ALTO.xml'), 'text/xml'),
+             'PAGE_0017_PAGE.xml': (os.path.join(FOLDER_KANT, 'data', 'OCR-D-GT-PAGE', 'PAGE_0017_PAGE.xml'), 'text/xml'),
+             'PAGE_0020_PAGE.xml': (os.path.join(FOLDER_KANT, 'data', 'OCR-D-GT-PAGE', 'PAGE_0020_PAGE.xml'), 'text/xml'),
+             }
+
+
+def _get_kant_data(key):
+    if key in DATA_KANT.keys():
+        (path, mime) = DATA_KANT[key]
+        with open(path, mode='rb') as _file:
+            return (_file.read(), mime)
+
+
+def request_behavior(*args):
```

Comment:
This is also a candidate for reuse in a dedicated ocrd_testing package https://github.com/OCR-D/core/issues/587